src/ol/render/webgl/webglimmediate.js

```diff
@@ -132,52 +163,35 @@ ol.render.webgl.Immediate.prototype.drawFeature = function(feature, style) {
     zIndex = 0;
   }
   this.drawAsync(zIndex, function(render) {
-    render.setFillStrokeStyle(style.getFill(), style.getStroke());
-    render.setImageStyle(style.getImage());
-    render.setTextStyle(style.getText());
-    var type = geometry.getType();
-    var renderGeometry = ol.render.webgl.Immediate.GEOMETRY_RENDERERS_[type];
-    // Do not assert since all kinds of geometries are not handled yet.
-    // In spite, render what we support.
-    if (renderGeometry) {
-      renderGeometry.call(render, geometry, null);
-    }
+    render.setStyle(style);
+    goog.asserts.assert(geometry);
```

Comment:
I'll add one.  I find the line numbers more useful.  But I know the message can be comforting.